File: src/test/java/edu/hm/hafner/analysis/IssueSoftAssertions.java
package edu.hm.hafner.analysis;

import edu.hm.hafner.analysis.assertj.SoftAssertions;

import java.util.function.Consumer;

public class IssueSoftAssertions extends SoftAssertions{
    public IssueAssert assertThat(Issue actual) {
        return proxy(IssueAssert.class, Issue.class, actual);
    }

    /*
    public static void assertSoftlyIssue(Consumer<IssueSoftAssertions> softly) {
        IssueSoftAssertions assertions = new IssueSoftAssertions();
        softly.accept(assertions);
        assertions.assertAll();
    }*/
}

File: src/test/java/edu/hm/hafner/analysis/IssueSoftAssertions.kt
package edu.hm.hafner.analysis

import edu.hm.hafner.analysis.assertj.SoftAssertions

open class IssueSoftAssertions : SoftAssertions() {
    fun assertThat(actual: Issue): IssueAssert {
        return proxy(IssueAssert::class.java, Issue::class.java, actual)
    }
}
